src/main/java/frc/robot/Constants.java

```diff
@@ -596,4 +596,39 @@ public static final class CraneConstants {
     public static final Translation2d kPositionIntake =
       new Translation2d(Units.degreesToRadians(32.042), 0.6824);
   }
+
+  public static final class ClimberConstants {
+    public static final boolean kEnable = false;
+    public static final int kMotorID = 0; //TODO: Get motor ID.
+    public static final double kHardMin = 0.0; //TODO: Set.
+    public static final double kMaxSpeed = 0.5; //TODO: Set. This number is from ACDC.
+    public static final double kMaxExtendedAngle = 90; //TODO: Set.
```

Comment:
Let's change `90` to `Math.toRadians(90)` (and same for the other angle constants) so that we don't forget the conversion during tuning.